.github/workflows/rdev-update-for-pr.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches-ignore:
       - prod
     types:
-      - open
+      - opened
```

Comment:
@Bento007 I think this was a bug that prevents rdevs from being created for single-commit new PR's. The rdev workflow was not triggered for me.

Reply:
close and reopen will fix it